jsonxx.h

```diff
@@ -195,6 +195,20 @@ namespace jsonxx {
         ~Value() { reset(); }
         void reset();
 
+#ifdef JSONXX_ALLOW_INVALID_TYPES
+        template<typename T>
+        [[deprecated("this type is not natively supported by jsonxx, therefore its value will be converted to 'null'")]]
+        void import( const T& t ) {
+            reset();
+            type_ = INVALID_;
```

Comment:
`JSONXX_ALLOW_INVALID_TYPES` should only be used in tests, like in jsonxx_test.cc